corehq/apps/groups/tests/test_groups.py

```diff
@@ -49,6 +52,31 @@ def _check_all_users(userlist):
         _check_all_users([u._id for u in group.get_static_users(is_active=False)])
         _check_all_users(group.get_static_user_ids(is_active=False))
 
+    def test_remove_users(self):
+        group = Group(domain=DOMAIN, name='group',
+                      users=[self.active_user._id, self.inactive_user._id, self.deleted_user._id])
+        group.save()
+
+        group.remove_user(self.active_user._id)
+        removal_record = UserRemoval.objects.get(group_id=group.get_id, user_id=self.active_user._id)
+        self.assertIsNotNone(removal_record)
+
+        group.add_user(self.active_user._id)
+        group.remove_user(self.active_user._id)
+        removal_record_updated = UserRemoval.objects.get(group_id=group.get_id, user_id=self.active_user._id)
+        self.assertNotEqual(removal_record.removed_on, removal_record_updated.removed_on)
+
+    def test_remove_multiple_users(self):
+        group = Group(domain=DOMAIN, name='group',
+                      users=[self.active_user._id, self.inactive_user._id, self.deleted_user._id])
+        group.save()
+
+        self.assertEqual(len(list(UserRemoval.objects.filter(group_id=group.get_id))), 0)
+        group.remove_user(self.active_user._id, save=False)
+        group.remove_user(self.inactive_user._id, save=False)
```

Comment:
might be nice to add a line here to test that nothing happens until you call save:
```python
self.assertEqual(len(list(UserRemoval.objects.filter(group_id=group.get_id))), 0)
```